app/views/annual_leave_requests/new.html.erb

```diff
@@ -0,0 +1,74 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/back_link", {
+      href: "javascript:window.history.back()"
+    } %>
+
+    <% if @annual_leave_request.errors.any? %>
+      <%= render "govuk_publishing_components/components/error_summary", {
+        title: "There was a problem with your annual leave request",
+        description: "Fix the following issue(s) and re-submit your request",
+        items: error_messages_for(@annual_leave_request)
+      } %>
+    <% end %>
+
+    <%= render "govuk_publishing_components/components/title", {
+      title: "Request annual leave",
+      average_title_length: "long",
+    } %>
+
+    <%= form_with url: annual_leave_requests_check_path, method: :get do %>
+      <%= render "govuk_publishing_components/components/date_input", {
+        legend_text: "Date from:",
+        name: "annual_leave_request[date_from]",
+        hint: "For example, 31 3 2023. Dates are inclusive.",
+        items: [
+            {
+              name: "day",
+              width: 2,
+              value: @annual_leave_request.date_from&.day
+            },
+            {
+              name: "month",
+              width: 2,
+              value: @annual_leave_request.date_from&.month
+            },
+            {
+              name: "year",
+              width: 4,
+              value: @annual_leave_request.date_from&.year
+            }
+          ],
+        error_message: error_message_for_input(@annual_leave_request.errors, :date_from)
+      } %>
```

Comment:
Should these form fields have separate labels? I notice on the design spec it looks a bit different - there's an extra heading or label above the legend. I'm not that knowledgeable about the design system though so not sure which is correct.